webpack.common.js

```diff
@@ -8,6 +8,29 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const { DefinePlugin, IgnorePlugin } = require('webpack');
 const packageJson = require('./package.json');
 
+const packageConfig = require('./package.json');
```

Comment:
why not reuse the packageJson include ?

Reply:
You are right, I just copied the whole code over without checking for duplicate imports. Fixed.